app/src/main/java/com/example/myweatherapp/viewHolders/NewsHolder.kt

```diff
@@ -7,25 +7,31 @@ import com.example.myweatherapp.R
 import com.example.myweatherapp.databinding.NewsHolderBinding
 import com.example.myweatherapp.interfaces.OnItemClickListener
 import com.example.myweatherapp.models.NewsModel
+import java.time.format.DateTimeFormatter
 
 
 class NewsHolder(private val newsListener: OnItemClickListener, itemView: View) : RecyclerView.ViewHolder(itemView) {
 
     private val binding = NewsHolderBinding.bind(itemView)
 
-    fun bind(news: NewsModel) = with(binding) {
+    fun bind(news: NewsModel, favoriteUrls: List<String>) = with(binding) {
+        val dateTimeFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy")
         newsTitle.text = news.newsTitle
         newDescription.text = news.newsDescription
         newsAuthor.text = news.newsAuthor
-        newsDate.text = news.newsPublishedAt
+        newsDate.text = dateTimeFormatter.format(news.newsPublishedAt)
         Glide
             .with(binding.root.context)
             .load(news.newsUrlToImage)
             .thumbnail(0.1f)
             .centerInside()
-            /*.override(400, 200)*/
             .into(newsImage)
 
+        if (favoriteUrls.contains(news.newsUrl)) {
```

Comment:
can't you just add some flag like `isFavorite` into the `NewsModel` class and use it here instead of having that `favoriteUrls` array?